reconstruct/reconstruct_test.go

```diff
@@ -31,18 +33,59 @@ func Test_reconstruct(t *testing.T) {
 
 		tr, err := expandedTree.ReadTree(test.newickFilename, "leavesOnly.fasta")
 		exception.FatalOnErr(err)
-		leaves := expandedTree.GetLeaves(tr)
+		leaves = expandedTree.GetLeaves(tr)
 		for i := 0; i < len(leaves[0].Fasta.Seq); i++ {
 			LoopNodes(tr, i)
 		}
 		WriteTreeToFasta(tr, "reconOut.fasta")
-		accuracyData := ReconAccuracy("simOut.fasta", "reconOut.fasta", "leavesOnly.fasta", "testdata/genePred.gp")
+		accuracyData, _ = ReconAccuracy("simOut.fasta", "reconOut.fasta", "leavesOnly.fasta", "testdata/genePred.gp", false)
 		for name, accuracy := range accuracyData {
 			log.Printf("%s %f \n", name, accuracy)
 		}
 	}
-	fileio.EasyRemove("reconOut.fasta")
-	fileio.EasyRemove("simOut.fasta")
+
+	if accuracyData[leaves[0].Name] != 100 {
```

Comment:
This looks like an "if" with nothing inside it (so it could just be removed).  Is there supposed to be something in it?